docs/developer-docs/smart-contracts/maintain/settings.mdx

```diff
@@ -9,162 +9,87 @@ import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 
 # Settings
 
-<MarkdownChipRow labels={["Beginner"]} />
+<MarkdownChipRow labels={["Beginner", "Tutorial"]} />
 
 ## Overview
 
-Each <GlossaryTooltip>canister</GlossaryTooltip> has a group of possible settings that control its behavior. Only a controller of the canister can read and modify its settings. Currently, there are seven canister setting fields:
+Each <GlossaryTooltip>canister</GlossaryTooltip> has a group of possible settings that control its behavior.
+
+:::caution
+Only a controller of the canister can read and modify a canister's settings.
+:::
+
+Currently, there are seven canister setting fields:
 
 1. `controllers`: The list of controllers of the canister.
-1. `compute_allocation`: Amount of compute that this canister has allocated.
-1. `memory_allocation`: Amount of memory (storage) that this canister has allocated.
-1. `freezing_threshold`: A safety threshold to protect against canister deletion due to running out of cycles.
+1. `compute_allocation`: Amount of compute that the canister has allocated.
+1. `memory_allocation`: Amount of memory (storage) that the canister has allocated.
+1. `freezing_threshold`: A safety threshold to prevent the canister from running out of cycles and avoid being deleted.
 1. `reserved_cycles_limit`: A safety threshold to protect against spending too many cycles for resource reservation.
 1. `wasm_memory_limit`: A safety threshold to protect against reaching the 4GiB hard limit of 32-bit Wasm memory.
 1. `log_visibility`: Controls who can read the canister logs.
 
-## Viewing a canister's current settings
+## Viewing current settings
 
 There are two ways to read the settings of a canister:
 
-1. Command line: `dfx canister status <canister-id>`. Note that this command returns canister settings together with other canister status information.
-1. Programmatically: Call the `canister_status` endpoint of the IC management canister.  The canister settings will be in the `settings` field of the result.
+1. Command line: [`dfx canister status <canister-name>`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-status). Note that this command returns the canister's settings, along with other canister status information.
+2. Programmatically: Call the [`canister_status`](https://internetcomputer.org/docs/current/references/ic-interface-spec) endpoint of the IC management canister. The canister's settings will be in the `settings` field of the result.
 
-Note that only a controller of the canister can read canister settings.
-
-An example of using `dfx` in a command line:
+### Command line
 
 ```bash
-> dfx canister status ajuq4-ruaaa-aaaaa-qaaga-cai
+dfx canister status ajuq4-ruaaa-aaaaa-qaaga-cai
+```
+
+<details>
+<summary>Output</summary>
+```
 Canister status call result for ajuq4-ruaaa-aaaaa-qaaga-cai.
```

Comment:
Do you need to replace the canister ID and principals with a generic like <canister_name> and <principal_id> or make it more clear that the output should look like something like this?